### PR TITLE
fix: Add dropdown menu to support all OVH providers

### PR DESCRIPTION
**Pull request by dohseven**
As discussed in issue #14, I propose a dropdown menu in order to be able to select each OVH provider.

### CODE REVIEW

**Comment by lmorel3** [comment]
Des nouvelles de cette PR ? :)

**Comment by dohseven** [comment]
> Des nouvelles de cette PR ? :)

Chez moi ça marche :wink: 
J'imagine qu'il faut demander à @LucsT si Cozy veut bien l'intégrer ou non.

File: manifest.konnector [hunk 1]
```diff
@@ -12,6 +12,47 @@
     "online_services"
   ],
   "fields": {
+    "provider": {
+      "type": "dropdown",
+      "default": {
```
Comment (doubleface) [inline manifest.konnector]: This attribute does not work with the last cozy-home. Please remove it.

Comment (doubleface) [reply manifest.konnector]: You still have some code to handle default value in the connector itself

Comment (dohseven) [reply manifest.konnector]: @doubleface You're right, the default value is already handled in the konnector. I removed the "default" attribute.

**Pull request merged**